vector/src/main/java/im/vector/app/features/voicebroadcast/usecase/StartVoiceBroadcastUseCase.kt

```diff
@@ -81,9 +81,11 @@ class StartVoiceBroadcastUseCase @Inject constructor(
     }
 
     private fun startRecording(room: Room, eventId: String, chunkLength: Int) {
-        voiceBroadcastRecorder?.listener = VoiceBroadcastRecorder.Listener { file, sequence ->
-            sendVoiceFile(room, file, eventId, sequence)
-        }
+        voiceBroadcastRecorder?.addListener(object : VoiceBroadcastRecorder.Listener {
+            override fun onVoiceMessageCreated(file: File, sequence: Int) {
+                sendVoiceFile(room, file, eventId, sequence)
+            }
+        })
```

Comment:
You need to remove the listener somewhere.

Reply:
I did not remove it from here because it will be removed when the record will be stopped from StopVoiceBroadcastUseCase (stopping a record cause the cleaning of the listeners).